.github/workflows/continuous-build.yml

```diff
@@ -21,20 +21,31 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.9
-    - name: Install dependencies.
+    - name: Start Test Harness
       run: |
-        cp app/* . -R
-        rm -rf app
+       chmod u+x manage
+        ./manage up
+       echo waiting for dockers to startup, sleeping for 60 seconds
+       sleep 60
+       echo wait over, continuing
+    - name: Install test dependencies.
+      run: |
+        cd app
         python -m pip install --upgrade pip
-        pip install pytest
+        pip install pytest pytest-custom_exit_code
```

Comment:
this is relevant as other wise pytest exits with 1 when tests fail so the build does not continue - we want the build to continue as we want sonar to run even when tests fail.